change/@internal-react-composites-1c279933-1871-47f9-a9ad-579ebca2587d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Added changes to see file specific upload errors in the errorbar above Sendbox",
```

Comment:
likewise here, gear the comment towards the package - in this case functionality is being added to the chat and callwithchat composites; "Show file upload errors above sendbox in the Composites containing Chat features"

Reply:
Done